hmf/density_field/transfer_models.py

```diff
@@ -713,3 +718,65 @@ class EH(EH_BAO):
     """Alias of :class:`EH_BAO`."""
 
     pass
+
+
+class CLASS(FromFile):
+    """
+    Transfer function computed by CLASS.
+
+    Parameters
+    ----------
+
+    cosmo : :class:`astropy.cosmology.FLRW` instance
+        The cosmology used in the calculation
+
+    \*\*model_parameters : unpack-dict
+        Parameters specific to this model.
+
+        **class_params:**
+            Dict of params to set for CLASS object
+        **class_object:**
+            Custom class object. If not set, a new Class object will be
+            created.
+        **flip_t:**
```

Comment:
The transfer function in hmf is *only* used to get the power spectrum (at this point). I think it's reasonable just to always take the absolute for now. It might be good to post an issue about this though, and we can think about returning the actual transfer function rather than the log. I chose the log because at the time I thought it must always be positive, and the log is easier to take a spline of.

Reply:
Ah, you'll need to remove `flip_t` from the docsring as well :-)